frontend/src/routes/[[lang=locale]]/candidate/(protected)/questions/[questionId]/+page.svelte

```diff
@@ -112,21 +106,21 @@
         answerStore[questionId] = {
           id: answer.id,
           key: previousLikert,
-          openAnswer: toLocalizedString(openAnswer)
+          openAnswer: openAnswer
         };
```

Comment:
There is some duplicate code here. Do you think it could be avoided?

Reply:
The two parts are quite similar but there are small differences depending on whether a new answer is saved or an old one updated.